pkg/inspector/image-inspector_ginkgo_test.go

```diff
@@ -0,0 +1,171 @@
+// +build integrationtest
+
+package inspector_test
+
+import (
+	"fmt"
+	"github.com/fsouza/go-dockerclient"
+	. "github.com/onsi/ginkgo"
```

Comment:
ginkgo by convention uses the . import (this file's skeleton is generated by `ginkgo generate`)

look at https://github.com/openshift/origin/blob/master/vendor/k8s.io/kubernetes/test/e2e/framework/pods.go#L36 as an example of what I mean

So I think the intention for Ginkgo is to be used a little differently (I suppose because it's only used for testing)